### PR TITLE
Remove empty lines after rewrite

### LINKED ISSUE #119: JavaDoc: RemoveJavaDocAuthorTag and RemoveEmptyJavaDocParameters should remove empty JavaDoc after cleanup
```diff
 /**
- * @author jzieren
  */
```

Expected:

```diff
- /**
-  * @author jzieren
-  */
```

Similar for `RemoveEmptyJavaDocParameters` - and thus refs https://github.com/openrewrite/rewrite-static-analysis/issues/98.

Maybe both `RemoveJavaDocAuthorTag` and `RemoveEmptyJavaDocParameters` should remove empty lines and empty JavaDoc after cleanup.

### PR DESCRIPTION
**Pull request by koppor**
- Fixes #119 
- Fixes https://github.com/openrewrite/rewrite-static-analysis/issues/98

### Checklist

- [x] I've added unit tests to cover both positive and negative cases
- [ ] I've read and applied the [recipe conventions and best practices](https://docs.openrewrite.org/authoring-recipes/recipe-conventions-and-best-practices)
- [ ] I've used the IntelliJ IDEA auto-formatter on affected files

### CODE REVIEW

**Comment by koppor** [comment]
I left all commits, because there will be a squash-merge, won't it?

**Comment by koppor** [comment]
I did not see https://github.com/openrewrite/rewrite-static-analysis/pull/190 while implementing. Think, my code is shorter then the one suggested at https://github.com/openrewrite/rewrite-static-analysis/pull/190.

File: src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java [hunk 1]
```diff
@@ -357,14 +404,6 @@ void singleLineReturnNoSpace() {
...
                 """
                   class Test {
-                      /**@return*/
-                      int method() {
-                      }
-                  }
-                  """,
-                """
-                  class Test {
-                      /***/
```
Comment (timtebeek) [inline src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java:367]: With these lines removed the test no longer verifies anything; is this a case that was hard to cover? Should the test be removed instead?

File: src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java [hunk 2]
```diff
@@ -380,14 +419,6 @@ void singleLineThrowsNoSpace() {
...
                 """
                   class Test {
-                      /**@throws*/
-                      void method() throws IllegalStateException {
-                      }
-                  }
-                  """,
-                """
-                  class Test {
-                      /***/
```
Comment (timtebeek) [inline src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java:390]: Similar to above: we no longer seem to assert anything meaningful here; especially not in relation to the test name. Should the test be restored or removed?

**Review by timtebeek — COMMENTED** [review]
Thanks a lot for taking this on! #190 had indeed lingered as there were a lot of formatting changes, making it a challenge to review. That earlier PR did have explicit tests for `@throws` and `@return`, which seem to have been removed here. Should any of that make it in here?

I'm inclined to proceed with this PR, as I like the minimal approach you took at making changes, such that it's easier to spot what goes into removing empty lines. Let's get the comments resolved and get this fix in.

File: src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java (hunk 1 above)
Comment (koppor) [reply src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java]: TBH, I totally forgot, why I removed these. Maybe by accident?

**Pull request merged**